server/middlewares/UserValidation.js

```diff
@@ -212,18 +212,16 @@ class UserValidation {
     * @static
     */
   static validateUserLogin(req, res, next) {
-    const { email, password } = req.body;
+    const { email } = req.body;
     const errors = {};
     const returnError = () => res.status(422).json({
       errors
     });
-    if (email && password) return next();
-    if (!email && !password) {
+    if (email) return next();
+    if (!email) {
```

Comment:
Is this check not the same as line 224? If so, please take this out.

Reply:
I will fix this asap.